### PR TITLE
Fixes #32020 - Allow disabling foreman callback plugin

### PR DESCRIPTION
**Pull request by adamruzicka**
unless applying roles

requires:
- [x] https://github.com/theforeman/foreman-ansible-modules/pull/1161

### CODE REVIEW

File: lib/foreman_ansible_core/runner/ansible_runner.rb [hunk 1]
```diff
@@ -104,7 +106,9 @@ def write_playbook
       end
 
       def start_ansible_runner
-        command = ['ansible-runner', 'run', @root, '-p', 'playbook.yml']
+        env = {}
+        env['FOREMAN_DISABLE'] = '1' if @rex_command
+        command = [env, 'ansible-runner', 'run', @root, '-p', 'playbook.yml']
```
Comment (ezr-ondrej) [inline lib/foreman_ansible_core/runner/ansible_runner.rb:111]: Is this a correct way to set env variables? passing the env hash seems to be quite wild.

**Review by ezr-ondrej — COMMENTED** [review]
This doesn't seem to work, the rex_command is set and I'm quite sure I'm running the right version of callback.

File: lib/foreman_ansible_core/runner/ansible_runner.rb (hunk 1 above)
Comment (adamruzicka) [reply lib/foreman_ansible_core/runner/ansible_runner.rb:111]: Not sure if it is the correct way, but it is the way we did it for ansible-playbook

Comment (ezr-ondrej) [reply lib/foreman_ansible_core/runner/ansible_runner.rb:111]: Ignore me, I was using wrong callback :) It works... still looks weird, but works xD

**Review by ezr-ondrej — APPROVED** [review]
This works great, just small nits in the modules PR after testing :)

File: lib/foreman_ansible_core/runner/ansible_runner.rb [hunk 2]
```diff
@@ -104,7 +106,9 @@ def write_playbook
       end
 
       def start_ansible_runner
-        command = ['ansible-runner', 'run', @root, '-p', 'playbook.yml']
+        env = {}
+        env['FOREMAN_DISABLE'] = '1' if @rex_command
```
[the remark this answers is not in the document]
Comment (adamruzicka) [reply lib/foreman_ansible_core/runner/ansible_runner.rb]: Changed to `FOREMAN_CALLBACK_DISABLE`

**Review by ezr-ondrej — APPROVED** [review]
This will work only with next release of `foreman-ansible-modules` but this will not break without is, just won't work, so no need to wait for it :)

**Pull request merged**